srcpkgs/mg/template

```diff
@@ -1,29 +1,26 @@
 # Template file for 'mg'
 pkgname=mg
-version=20210314
+version=20220614
 revision=1
 hostmakedepends="pkg-config"
-makedepends="libbsd-devel ncurses-devel"
+makedepends="libbsd-devel libmd-devel ncurses-devel"
 short_desc="Micro GNU/emacs"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Public Domain"
 homepage="https://github.com/hboetes/mg"
 distfiles="https://github.com/hboetes/mg/archive/${version}.tar.gz"
-checksum=6820e76a0e27d9a744624d421b3b30b0030140c2a817df1310b25c3abe397106
+checksum=d3bc16baba82457c4f60ecd258762ee2ed848064f79729fe92e29bfd181af121
 
-CFLAGS="-DTCSASOFT=0"
+CFLAGS+=" -DTCSASOFT=0"
```

Comment:
this change isn't necessary

Reply:
Thanks, I reverted this change.  After looking more closely, I think the build and install functions can be removed by just using the gnu-makefile build style.